src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -21,27 +21,34 @@
  *******************************************************************************
  ******************************************************************************/
 
+#include "general-commissioning-server.h"
+
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app-common/zap-generated/cluster-objects.h>
+#include <app/AppConfig.h>
 #include <app/AttributeAccessInterfaceRegistry.h>
-#include <app/CommandHandler.h>
```

Comment:
Why is it OK to remove the CommandHandler, ConcreteCommandPath, CommissioningWindowManager includes?  This file definitely uses those types, and relying on some other random header to include them seems odd.  Please put them back.

Reply:
Added them back. kept removal of ember attribute storage and span (did not see span usage)